src/Lesson_6/TaskOne/CreditCard.java

```diff
@@ -0,0 +1,35 @@
+package Lesson_6.TaskOne;
+
+
+public class CreditCard {
+    private final long id;
+    private String accountNumber;
+    private double currentAccountAmount;
+
+    public void setCurrentAccountAmount(double currentAccountAmount) {
```

Comment:
why do we need this setter, if it is not in use in constructor?

Reply:
I don't initialize currentAccountAmount in constructor because the user needs to set this value from console. Should this setter be used in this constructor? How can the value be added via console in this case if this setter is used only when creating card?